staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -115,17 +115,14 @@ type eventBroadcasterImpl struct {
 // The return value can be ignored or used to stop recording, if desired.
 // TODO: make me an object with parameterizable queue length and retry interval
 func (eventBroadcaster *eventBroadcasterImpl) StartRecordingToSink(sink EventSink) watch.Interface {
-	// The default math/rand package functions aren't thread safe, so create a
```

Comment:
I don't think this is true.

Reply:
It's not true now and probably wasn't true when I wrote this comment, either, but it was true in very early versions of go.